tests/unit/processor/calculator/test_calculator.py

```diff
@@ -158,6 +158,19 @@
         {"duration": "0.01"},
         {"duration": 10000.0},
     ),
+    (
+        "Ignore missing source fields",
+        {
+            "filter": "duration",
+            "calculator": {
+                "calc": "${missing_field} * 10e5",
+                "target_field": "duration",
+                "ignore_missing_fields": True,
+            },
+        },
+        {"duration": "0.01"},
+        {"duration": "0.01"},
+    ),
```

Comment:
I think the calculator `ignore_missing_fields` option needs a separate documentation, because it would be good to know what the actual behavior is, if only one field is missing, but another one exists. As soon as one field is missing the whole calculation is aborted, which makes sense, but maybe it's good to let the user know about that in the documentation.

Reply:
done